### ISSUE TITLE
When trying to install I get these three errors.

### ISSUE THREAD

**Issue by zxdev** [eiannone/keyboard#7]
src/github.com/eiannone/keyboard/keyboard.go:161:17: undefined: unix.TCGETS
src/github.com/eiannone/keyboard/keyboard.go:177:17: undefined: unix.TCSETS
src/github.com/eiannone/keyboard/keyboard.go:211:11: undefined: unix.TCSETS

I can't seems to find any reference to these elsewhere in the package, so it seems like something is missing from the package.

**Comment by eiannone** [comment]
It seems that the package `golang.org/x/sys/unix` is not installed in your system.
Did you install with `go get`? (`go get -u github.com/eiannone/keyboard`)
What do you get if you run `go list golang.org/x/sys/unix` ?

**Comment by zxdev** [comment]
Okay, so here is what I have done, but still no joy. 

**The TL;DR is that seems like this has been removed or refactored in the /x/unix package which breaks your package**

MacBook:golang.org Mac$ go list golang.org/x/sys/unix
golang.org/x/sys/unix

. . . seems like it is there . . .

go get -u github.com/eiannone/keyboard
 cd /Users/Mac/go/src/golang.org/x/sys; git pull --ff-only
error: Your local changes to the following files would be overwritten by merge:
        unix/creds_test.go
        unix/flock.go
 . . .
       windows/ztypes_windows_386.go
       windows/ztypes_windows_amd64.go
Please commit your changes or stash them before you merge.
Aborting
Updating 94b7606..062cd7e
package golang.org/x/sys/unix: exit status 1

 . . . so there were 284 changes, so I did a commit so it would be happy . . .

MacBook:golang.org Mac$ go get -u github.com/eiannone/keyboard
 cd /Users/Mac/go/src/golang.org/x/sys; git pull --ff-only
fatal: Not possible to fast-forward, aborting.
package golang.org/x/sys/unix: exit status 128

  . . . huh ?? . . .

So I renamed golang.org/x/sys to golang.org/x/sys.v0913 and then do the go get again. This time the golang.org/x/sys was recreated, but still no joy . . . 

MacBook:golang.org Mac$ go get -u github.com/eiannone/keyboard
 github.com/eiannone/keyboard
../github.com/eiannone/keyboard/keyboard.go:161:17: undefined: unix.TCGETS
../github.com/eiannone/keyboard/keyboard.go:177:17: undefined: unix.TCSETS
../github.com/eiannone/keyboard/keyboard.go:211:11: undefined: unix.TCSETS
MacBook:golang.org Mac

So I then renamed the entire go directory and started over, created a new directory that was complete empty, and then tried again...

MacBook:~ Mac$ mv go go.v0913
MacBook:~ Mac$ mkdir go
MacBook:~ Mac$ cd go
MacBook:go Mac$ mkdir src pkg bin
MacBook:go Mac$ go get -u github.com/eiannone/keyboard
 github.com/eiannone/keyboard
github.com/eiannone/keyboard/keyboard.go:161:17: undefined: unix.TCGETS
github.com/eiannone/keyboard/keyboard.go:177:17: undefined: unix.TCSETS
github.com/eiannone/keyboard/keyboard.go:211:11: undefined: unix.TCSETS

 . . . checking directories shows all I have is . . .

MacBook:src Mac$ ls *
github.com:
eiannone

golang.org:
x


**So it seems like it has been removed or refactored in the /x/unix package, as best as I can tell.**

That is what I did, can you duplicate the issue with a clean/fresh install?

**Comment by eiannone** [comment]
After some investigation I realized that TCSETS constant is not included in `golang.org/x/sys/unix` package for the _darwin_ operating system (macOS).
I will try to find a workaround...

**Comment by eiannone** [comment]
I've just added the missing constants for _darwin_ and other platforms. Currently I have no access to a macOS system so I cannot verify if it works.
@g-w-p could you please update to the latest commit and let me know if you are able to compile now?
Thank you.

**Comment by zxdev** [comment]
It works now.
